app/controllers/accounts_controller.rb

```diff
@@ -1,2 +1,31 @@
 class AccountsController < ApplicationController
+  def index
+    @accounts = Account.where(user_id: current_user.id)
+  end
+
+  def show
+    @account = Account.find(params[:id])
+  end
+
+  def new
+    @account = Account.new
+  end
+  
+  def create
+    @account = Account.new(account_strong_params)
+    @account.user_id = current_user.id
+    @account.save ? (redirect_to account_path(@account)) : (render :new)
+  end
+
+  def destroy
+    @account = Account.find(params[:id])
+    @account.destroy
+    redirect_to accounts_path
+  end
+  
+  def account_strong_params
+    params.require(:account).permit(
```

Comment:
you can have
`@account = ...`
and then add the whole thing in a `before_action`

Reply:
+ `private` for that last one 😄